UI_DSM.Server/Services/SearchService/SearchService.cs

```diff
@@ -0,0 +1,139 @@
+﻿// --------------------------------------------------------------------------------------------------------
+// <copyright file="SearchService.cs" company="RHEA System S.A.">
+//  Copyright (c) 2022 RHEA System S.A.
+// 
+//  Author: Antoine Théate, Sam Gerené, Alex Vorobiev, Alexander van Delft, Martin Risseeuw, Nabil Abbar
+// 
+//  This file is part of UI-DSM.
+//  The UI-DSM web application is used to review an ECSS-E-TM-10-25 model.
+// 
+//  The UI-DSM application is provided to the community under the Apache License 2.0.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------
+
+namespace UI_DSM.Server.Services.SearchService
+{
+    using System.Text;
+    using System.Text.Json;
+
+    using CDP4Common.CommonData;
+    using CDP4Common.EngineeringModelData;
+
+    using CometSearch.Common;
+
+    using GP.SearchService.SDK.Definitions;
+
+    using Microsoft.AspNetCore.WebUtilities;
+
+    using UI_DSM.Client.Extensions;
+
+    /// <summary>
+    ///     Service that enable the indexing and searching of UI-DSM data also than 10-25 data
+    /// </summary>
+    public class SearchService : ISearchService
```

Comment:
Neat! That's the kind of client that I had in mind when I was building the search service